includes/REST_API/Embed_Controller.php

```diff
@@ -293,10 +317,27 @@ private function url_to_post( $url ) {
 			// url_to_postid() does not recognize plain permalinks like https://example.com/?web-story=my-story.
 			// Let's check for that ourselves.
 
-			$url_host      = str_replace( 'www.', '', wp_parse_url( $url, PHP_URL_HOST ) );
-			$home_url_host = str_replace( 'www.', '', wp_parse_url( home_url(), PHP_URL_HOST ) );
+			/**
+			 * The URL's hostname.
+			 *
+			 * @var string|false|null $url_host
+			 */
+			$url_host = wp_parse_url( $url, PHP_URL_HOST );
+			if ( $url_host ) {
+				$url_host = str_replace( 'www.', '', $url_host );
+			}
```

Comment:
These lines are repeated. Worth creating a little method on the class.

Reply:
A method for a 1 line `str_replace` call?